app/src/main/java/org/oppia/android/app/testing/DragDropTestFragment.kt

```diff
@@ -10,14 +10,18 @@ import org.oppia.android.app.fragment.FragmentComponentImpl
 import org.oppia.android.app.fragment.InjectableFragment
 import org.oppia.android.app.recyclerview.OnDragEndedListener
 import org.oppia.android.app.recyclerview.OnItemDragListener
-import org.oppia.android.app.story.StoryFragment
 import javax.inject.Inject
 
-/** Fragment for displaying a DragDropTestFragment. */
+/**
+ * Fragment for displaying a [DragDropTestFragment].
+ * Added to enable access to [BindableAdapter]
+ * which should be provided through dependency injection,
+ * this is replacing use case of [DragDropTestActivity] since adapter can not be injected to activities.
+ */
```

Comment:
```suggestion
/** Test-only fragment used for verifying ``BindableAdapter`` functionality. */
```

Simplification--I don't think we need to communicate as much here.

Reply:
Noted